templates/vue/src/components/modals/NodeModal/forms/ContentForm/MultipleChoiceForm.vue

```diff
@@ -0,0 +1,231 @@
+    :multipleAnswerSelected="multipleAnswerSelected"
+    :useImages="useImages"
+    :removeButtonDisabled="isRemoveButtonDisabled"
+    v-on:remove="choiceRows.splice(index,1)" 
+    @newCheckBoxValue="handleNewCheckBoxValue">
+    </choice-row>
+        </b-form-checkbox-group>
+      </SortableList>
+    <b-button class="addButton" v-on:click="addNewChoice" variant="primary" squared>Add a choice</b-button>
+    <!-- <div class="mt-3">SelectedCheckBoxForm: <strong>{{ selectedCheckBoxForm }}</strong></div> -->
+      </b-form-group>
+
+      <b-form-group v-else-if="!multipleAnswerSelected">
+      <SortableList lockAxis="y" v-model="choiceRowsRadio" :useDragHandle="true" @input="updateOrderingRadioArray">
+        <b-form-checkbox-group v-model="selectedRadioForm">
+    <choice-row 
+    v-for="(choiceRow, index) in choiceRowsRadio"
+    :data-qa="`choicerow-radio-${choiceRow.id}`"
+    :item="choiceRow"
+    :key="choiceRow.id"
+    placeholder="answer option text"
+    :index="index"
+    :node="node" 
+    :multipleChoiceSelected="multipleChoiceSelected"
+    :multipleAnswerSelected="multipleAnswerSelected"
+    :useImages="useImages"
+    :isDisabled="selectedRadioForm.length > 0"
+    :selectedRadioChoice="selectedRadioForm[0]"
+    :removeButtonDisabled="isRemoveButtonDisabled"
+    v-on:remove="choiceRowsRadio.splice(index,1)" 
+    @newRadioValue="handleNewRadioValue">
+    </choice-row>
+    </b-form-checkbox-group>
+      </SortableList>
+    <b-button class="addButton" v-on:click="addNewChoiceRadio" variant="primary" squared>Add a choice</b-button>
+    <!-- <div class="mt-3">SelectedRadioForm: <strong>{{ selectedRadioForm }}</strong></div> -->
+      </b-form-group>
+  </b-form-group>
+</div>
+</template>
+
+<script>
+import ChoiceRow from "./ChoiceRow.vue"
+import FileUpload from "@/components/modals/common/FileUpload"
+import { ContainerMixin,} from 'vue-slicksort';
+const SortableList = {
+  mixins: [ContainerMixin],
+  template: `
+    <ul class="list">
+      <slot />
+    </ul>
+  `,
+};
+
+export default {
+  components: {
+  ChoiceRow,
+  FileUpload,
+  SortableList
+  },
+  props: {
+    node: {
+      type: Object,
+      required: true,
+    },
+    multipleChoiceSelected: {
+      type: Boolean,
+      required: true,
+    },
+    multipleAnswerSelected: {
+      type: Boolean,
+      required: true,
+    },
+  },
+  data() {
+    return {
+      selectedCheckBoxForm: [],
+      selectedRadioForm: [],
+      useImages: false,
+      choiceRows: [
+        {
+          id: 1,
+          imageurl: '',
+          value: '',
+          title: '1st Choice',
+        },
+        {
+          id: 2,
+          imageurl: '',
+          value: '',
+          title: '2nd Choice',
+        },
+        {
+          id: 3,
+          imageurl: '',
+          value: '',
+          title: '3rd Choice',
+        },
+      ],
+      nextChoiceRowId: 4,
+      newChoiceRowTitle: '',
+      choiceRowsRadio: [
```

Comment:
How come the choiceRowsRadio is an array? I'm not sure if I am understanding the code properly here, but this is the variable that tracks the _single_ selected answer for a radio button right? If a radio button only allows for a single selection, why is an array needed?

Reply:
no, that is what provides the data for the list of options that the author can put in values for radio, the variable that tracks the single selected answer for a radio button is selectedRadioForm 
<img width="594" alt="Screen Shot 2021-05-27 at 3 10 09 PM" src="https://user-images.githubusercontent.com/30706624/119903604-c8cfd880-befd-11eb-9e34-ae8dcb959272.png">